src/staticanalysis/bot/static_analysis_bot/tasks/coverity.py

```diff
@@ -154,7 +154,10 @@ def as_text(self):
         '''
         Build the text body published on reporters
         '''
-        return self.message
+        # If there is the reliability index use it
+        return f'Checker reliability (false positive risk) is {self.reliability.value}.\n{self.message}' \
```

Comment:
Are you sure you want to display the relaibility message before the message ?

Reply:
Yes, in order to be in ```your eye``` when you see that the patch that you authored has some issues.
We can change it later on.